### PR TITLE
Add versions.h

### PR DESCRIPTION
**Pull request by Dragorn421**
This was initially part of #2046 but I figured it was worthy of its own PR

### CODE REVIEW

**Comment by Dragorn421** [comment]
This also adds a macro for `OOT_NTSC_1_2`

**Review by cadmic — APPROVED** [review]

File: include/versions.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#ifndef VERSIONS_H
+#define VERSIONS_H
+
+// OOT versions in build order
+#define OOT_NTSC_1_2 1
+#define OOT_GC_JP 2
+#define OOT_GC_JP_MQ 3
+#define OOT_GC_US 4
```
Comment (fig02) [inline include/versions.h:8]: any issues with using an enum so we dont need to manually specify values?

Comment (cadmic) [reply include/versions.h:8]: we need the C preprocessor to be able to expand these, which makes me think it won't work as an enum but I haven't tried it

Comment (fig02) [reply include/versions.h:8]: I thought enums were processed by cpp, but after looking it up apparently im wrong

**Review by fig02 — APPROVED** [review]

**Pull request merged**